src/vendor-dashboard/layout/components/Sidebar.tsx

```
@@ -0,0 +1,565 @@
+                                                            key && collapsed
+                                                            ? 'text-white'
+                                                            : ''
+                                                    ) }
+                                                >
+                                                    { /* Render custom images or icons from the sidebar config */ }
+                                                    { item.icon &&
+                                                    item.icon.startsWith(
+                                                        '<img'
+                                                    ) ? (
+                                                        <span
+                                                            dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)